website/docs/getting-started/setup/instance-configuration/custom-domain/README.md

```diff
@@ -1,95 +1,75 @@
+## Prerequisites
+Before configuring SSL for your custom domain, make sure you have the following:
+1. A self-hosted Appsmith instance running. For more information about installing Appsmith, see [Installation Guides](/getting-started/setup/installation-guides)
+2. A domain name - You can get a custom domain from popular providers like [GoDaddy](https://in.godaddy.com/help/create-a-subdomain-4080), [Amazon Route 53](https://aws.amazon.com/premiumsupport/knowledge-center/create-subdomain-route-53/), [Digital Ocean](https://www.digitalocean.com/docs/networking/dns/how-to/add-subdomain/), [NameCheap](https://www.namecheap.com/support/knowledgebase/article.aspx/9776/2237/how-to-create-a-subdomain-for-my-domain), and [Domain.com](https://www.domain.com/help/article/domain-management-how-to-update-subdomains).
+3. Ports 80 and 443 are open and accessible.
 
-## Setting up SSL
+## Configure SSL
+Once you have a Custom Domain, you can choose to set up SSL in one of the following ways:
 
-SSL automatically set up for your instance once you configure a custom domain for your instance. All SSL certificates Appsmith creates are generated and kept up to date through [Let's Encrypt](https://letsencrypt.org).
+* [Generate and Configure SSL](#generate-and-configure-ssl)
+* [Configure a Custom SSL Certificate](#configure-a-custom-ssl)
 
-## Custom SSL certificate
 
-The container supports generating a free SSL certificate. If you have your own certificate, please follow these steps to use it inside the container.
+## Generate and configure SSL
+On Appsmith, you can choose to add your Custom Domain by using one of the following ways:
+* [Admin settings](#admin-settings)
+* [Environment Variables](#environment-variables)
 
-* Firstly, please rename your certificate file as `fullchain.pem` and key file as `privkey.pem`
-* Copy these files into the sub-directory `<mounting-directory>/ssl/` (_Note: Please change `<mounting-directory>` by the mounting volume directory in the `docker-compose.yml`. Default is `./stacks`_)
-* Restart the container using `docker restart appsmith`
-
-The container checks the certificate files in the folder `<mounting-directory>/ssl` and use them if they exist.
-
-:::note
-In case the certificate files have a different name from `fullchain.pem` and `privkey.pem`, it will fail to find the  custom certificate and auto-provisions the certificate by `Let's Encrypt`
-:::
+### Admin settings
+You can use Admin Settings User Interface to set up a custom domain. The **Custom Domain** property is available under **Advanced** Settings. Add the domain name to the Custom Domain setting and click the **SAVE & RESTART** button. The Custom domain setup can be done using the Custom Domain setting for all your installation types except Kubernetes. For more information about how to set up TLS on Kubernetes, see [How to configure TLS for Appsmith Kubernetes installation](/getting-started/setup/installation-guides/kubernetes#configure-tls).
```

Comment:
```suggestion
You can use [Admin Settings](link) page to set up a custom domain. The **Custom Domain** property is available under **Advanced** Settings. Add the domain name to the Custom Domain setting and click the **SAVE & RESTART** button. 

Should the below be a callout?
The Custom domain setup can be done using the Custom Domain setting for all your installation types except Kubernetes. For more information about how to set up TLS on Kubernetes, see [How to configure TLS for Appsmith Kubernetes installation](/getting-started/setup/installation-guides/kubernetes#configure-tls).
```

Reply:
Added a callout

Reply:
I've added a callout at the start of the page so that users don't have to read until the Admin settings section and realize they need to navigate to another page for Kubernetes. Also, I kept the para as is in case someone misses the callouts and comes here.